src/test/order_item_summary/OrderItemSummaryController.unit.test.ts

```diff
@@ -21,7 +26,27 @@ const request: any = {
                 }
             }
         }
-    },
+        }
+    }
+
+const generateMockRequest = (emailAddress: string, isSignedIn: number):Request => {
```

Comment:
Given this method is only used once here (so far anyway), it could perhaps be replaced with a separate (hardwired) object like `request` above. Just a thought.

Reply:
I might be using it for the basket link tickets, not sure though

Reply:
It is just a minor rationalisation.